application/features/bootstrap/SecureUserIntegrationContext.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Sil\Idp\IdSync\Behat\Context;
+
+use Sil\Idp\IdSync\common\components\adapters\SecureUserIdStore;
+use Sil\Idp\IdSync\common\models\User;
+use Sil\PhpEnv\Env;
+use yii\base\BaseObject;
+
+/**
+ * Defines application features from the specific context.
+ */
+class SecureUserIntegrationContext extends IdStoreIntegrationContextBase
+{
+    public function __construct()
+    {
+        echo 'Testing integration with Secure User.' . PHP_EOL;
+        parent::__construct();
+    }
+
+    /**
+     * @Given I can make authenticated calls to the ID Store
+     */
+    public function iCanMakeAuthenticatedCallsToTheIdStore()
+    {
+        $secureUserApiUrl = Env::requireEnv('TEST_SECURE_USER_CONFIG_apiUrl');
+        $secureUserApiKey = Env::requireEnv('TEST_SECURE_USER_CONFIG_apiKey');
+        $secureUserApiSecret = Env::requireEnv('TEST_SECURE_USER_CONFIG_apiSecret');
+
+        $this->idStore = new SecureUserIdStore([
+            'apiUrl' => $secureUserApiUrl,
+            'apiKey' => $secureUserApiKey,
+            'apiSecret' => $secureUserApiSecret,
+        ]);
+    }
+
+    /**
+     * @When I ask the ID Store for a specific active user
+     */
+    public function iAskTheIdStoreForASpecificActiveUser()
+    {
+        $this->activeEmployeeId = Env::requireEnv('TEST_SECURE_USER_EMPLOYEE_ID');
+        $this->result = $this->idStore->getActiveUser($this->activeEmployeeId);
```

Comment:
From what I can see, this method is only used for incremental sync. Do you still want to include it in the test? And maybe include a test for `getAllActiveUsers`?

Reply:
Since getActiveUser calls getAllActiveUsers, this is testing both in a way.  I didn't want the test to have to validate a long list of users. As long as one known user comes through, I assume that's good enough.